app/src/main/res/layout/activity_home.xml

```diff
@@ -6,4 +6,86 @@
     android:layout_height="match_parent"
     tools:context=".ui.home.HomeActivity">
 
+    <ScrollView
+        android:id="@+id/scrollView2"
```

Comment:
em cũng có thể đặt tên có ý nghĩa hơn thay vì để số 2 đằng sau :))

Reply:
Tại sao không để ScrollView là tằng to nhất bên ngoài mà phải gói trong 1 Constraint Layout nhỉ